lib/box2d/box2d_component.dart

```diff
@@ -134,6 +134,31 @@ abstract class BodyComponent extends Component {
 
   Vector2 get center => body.worldCenter;
 
+  void _renderChain(Canvas canvas, Fixture fixture) {
+    final ChainShape chainShape = fixtureDef.getShape();
+    final List<Vector2> vertices = Vec2Array().get(chainShape.getVertexCount());
+
+    for (int i = 0; i < chainShape.getVertexCount(); ++i) {
+      body.getWorldPointToOut(chainShape.getVertex(i), vertices[i]);
+      viewport.getWorldToScreen(vertices[i], vertices[i]);
+    }
+
+    final List<Offset> points = [];
+    for (int i = 0; i < chainShape.getVertexCount(); i++) {
+      points.add(Offset(vertices[i].x, vertices[i].y));
+    }
+
+    renderChain(canvas, points);
+  }
+
+  void  renderChain(Canvas canvas, List<Offset> points) {
+      final Paint paint = Paint()
```

Comment:
is there a way to override this colour? also, the default colour could be a static constant

Reply:
I just replicated the existing render functions. As it is just for viewing the bodies and a custom render function is required anyway, I figure this was ok. 

Example of existing below.

```
void renderCircle(Canvas canvas, Offset center, double radius) {
    final Paint paint = Paint()
      ..color = const Color.fromARGB(255, 255, 255, 255);
    canvas.drawCircle(center, radius, paint);
  }
```

Reply:
I see, makes sense